app/screens/channel/channel_base.js

```diff
@@ -255,13 +257,24 @@ export default class ChannelBase extends PureComponent {
             theme,
         } = this.props;
 
+        const errorTitle = {
```

Comment:
can we move this variables inside the `if (channelRequestFailed)` statement? no need to define them if they are not being used.

Reply:
Sure.